db/migrate/20151113173639_fix_book_author_relation.rb

```diff
@@ -0,0 +1,15 @@
+class FixBookAuthorRelation < ActiveRecord::Migration
+  def up
```

Comment:
@evan-007 why do we need to use `up` and `down` instead of `change` to let rails do reversible for us?